### PR TITLE
alba.co.jp

### PR DESCRIPTION
**Pull request by SKEIDs**
https://github.com/AdguardTeam/AdguardFilters/issues/124683 fixed only Screenshot 1

```
! https://www.alba.co.jp/sp/tour/news/article/no=185162
! Mobile
! Ad
alba.co.jp##.topbnrArea
! Social
alba.co.jp##.social-button
! https://www.alba.co.jp/tour/news/article/no=185162
! PC
! Ad
alba.co.jp###ad_rectangle03_04
```

<details><summary>screenshot</summary>

</details><br/>

### CODE REVIEW

**Review by Yuki2718 — CHANGES_REQUESTED** [review]

File: JapaneseFilter/sections/specific.txt [hunk 1]
```diff
@@ -4057,6 +4057,7 @@ sponichi.co.jp##[class^="banner"]
 golfnettv.com##.video-detail__ads
 sokuho.alba.co.jp##.lbadArea
 sokuho.alba.co.jp###tournamentBnrArea
+alba.co.jp###ad_rectangle03_04
```
[the remark this answers is not in the document]
Comment (SKEIDs) [reply JapaneseFilter/sections/specific.txt:4060]: ok,fixed

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
There are also popIn ad leftover if Annoyances filter is turned off. Add the domain to `#?#li > ._popIn_recommend_article_ad_reserved:upward(1)` in JPF.

**Review by Yuki2718 — CHANGES_REQUESTED** [review]
No I mean add to an existing rule.

**Comment by SKEIDs** [comment]
> No I mean add to an existing rule.
 I'm sorry. Fixed

**Review by Yuki2718 — APPROVED** [review]

**Review by zloyden — APPROVED** [review]

**Pull request merged**